installer/SystemConfiguration.php

```diff
@@ -191,15 +220,28 @@ public function createInstanceIdentifierChecksum($organizationName, $organizatio
     /**
      * Set the instance identifier checksum value
      * @param $organizationName
-     * @param $organizationEmail
-     * @param $randomNumber
+     * @param $email
```

Comment:
comment type is missing

Reply:
Didn't get this. Can you please explain more?

Reply:
I mean 
 @param string $email